superset-frontend/src/common/components/Modal.tsx

```diff
@@ -94,11 +108,13 @@ export default function Modal({
   disablePrimaryButton = false,
   onHide,
   onHandledPrimaryAction,
-  primaryButtonName,
+  primaryButtonName = 'OK',
```

Comment:
```suggestion
  primaryButtonName = t('OK'),
```
I'm having a _deja vu_ looking at some of this code... maybe this PR needs a rebase?